jerry-port/default/default-module.c

```diff
@@ -116,7 +116,7 @@ jerry_port_normalize_path (const char *in_path_p,   /**< input file path */
 {
   size_t ret = 0;
 
-#if defined (WIN32)
+#if defined (_WIN32)
```

Comment:
I suggest to have this change in a separate PR. Unfortunately, the code base does not use Windows-specific macros consistently. Preferrably a single commit should rewrite all `WIN32` occurrences to `_WIN32`.